perses/tests/test_geometry_engine.py

```diff
@@ -102,6 +103,8 @@ class FourAtomValenceTestSystem(GeometryTestSystem):
         Whether to include the angle force term
     torsion : Boolean, default True
         Whether to include the torsion force term
+    four_atom: Boolean, default True
```

Comment:
Get rid of all `Arguments` here---they should be under `__init__`.

Reply:
We can also change class name from `FourAtomValenceTestSystem` to `LinearChainValenceSystem`